src/pages/Search/components/Heatmap.styles.js

```diff
@@ -0,0 +1,77 @@
+import styled, { css } from 'styled-components';
+
+export const HeatmapContainer = styled.section`
+  width: 100%;
+  padding: 60px 163px 0 163px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+export const Grid = styled.div`
+  display: grid;
+  grid-template-columns: 154px minmax(auto, 960px);
+  grid-template-rows: 52px auto;
+`;
+
+export const DaysList = styled.div`
+  grid-column: 1 /2;
+  grid-row: 2 / -1;
+  background: ${({ theme }) => theme.colors.darkBlue};
+`;
+export const DayItem = styled.div`
+  line-height: 11px;
+  font-size: 15px;
+  height: 40px;
+  font-weight: 600;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-transform: capitalize;
+  ${({ theme }) => css`
+    color: ${theme.colors.grayLight};
+  `};
+`;
+
+export const HoursList = styled.div`
+  grid-column: 2/ -1;
+  display: flex;
+  ${({ theme }) => css`
+    background-image: linear-gradient(
+      to bottom,
+      ${theme.colors.gray2},
+      ${theme.colors.gray1}
+    );
+    border: 1px solid ${theme.colors.gray3};
+  `};
+`;
+export const HourItem = styled.div`
+  width: 80px;
+  padding: 17px 0;
+  font-weight: 500;
+  display: flex;
```

Comment:
Note: You don't need flex-box here. You could alternatively use `text-align: center` for horizontal and `line-height` (if the height is set of course